.changeset/twenty-suns-vanish.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/netlify': minor
+---
+
+Support for experimental redirects
+
+This adds support for the redirects RFC in the Netlify adapter. No changes are necessary, simply use configured redirects and the adapter will update your `_redirects` file.
```

Comment:
```suggestion
This adds support for the redirects RFC in the Netlify adapter, including a new `@astrojs/netlify/static` adapter for static sites. 

No changes are necessary when using SSR. Simply use configured redirects and the adapter will update your `_redirects` file.
```
Given our discussion, would updating the changeset something like this make sense? (And, if the Cloudflare one is new, too, something like this in its changeset also?)